test/get-unified-config.test.js

```diff
@@ -537,14 +537,62 @@ describe('get-unified-config', function () {
     // actually prevent the agent from running in a lambda environment, but the error will
     // be reported because the service key's format is invalid.
     //
-    it('an invalid serviceKey is a fatal error', function () {
+    it('an invalid serviceKey is a fatal error (output is masked)', function () {
       const serviceKey = `${'f'.repeat(32)}:service-name`
       process.env.APPOPTICS_SERVICE_KEY = serviceKey
 
       const cfg = guc()
 
       // the serviceKey is not valid and should be reported
-      const fatals = [`not a valid serviceKey: ${serviceKey}`]
+      const fatals = ['not a valid serviceKey: ffff...ffff:service-name']
+      const expected = Object.assign({ global: { serviceKey: '' }, fatals })
+      doChecks(cfg, expected)
+    })
+
+    it('an invalid serviceKey with no service name is a fatal error (output is masked)', function () {
+      const serviceKey = `${'f'.repeat(32)}`
+      process.env.APPOPTICS_SERVICE_KEY = serviceKey
+
+      const cfg = guc()
+
+      // the serviceKey is not valid and should be reported
+      const fatals = ['not a valid serviceKey: ffff...ffff:']
```

Comment:
For a service key like `fffffffffffffff`, appending the colon makes the assumption that it's the API Token and that service name is missing.  This is why PHP and Python just mask with something like `ffff...<invalid>`.